src/foam/u2/wizard/WizardletSection.js

```diff
@@ -79,6 +79,17 @@ foam.CLASS({
   ],
 
   methods: [
+    function activateProxySlot() {
+      var proxy$ = foam.core.ProxySlot.create();
+      this.isAvailable$ = proxy$;
+      var f = () => {
+        var isAvailable$ = this.section.createIsAvailableFor(this.data$);
```

Comment:
Could these two lines be combined?

Reply:
Shouldn't createIsAvailableFor(slot) be listening for slot changes? Why do we need to use the ProxySlot?

Reply:
I'll be honest, after several hours of debugging this was the best solution I could come up with. I have no idea why what you described doesn't work - I tried that first.

Reply:
It's possible that the slot gets broken. I tried adding `} catch (e) { console.error(e);` to the try blocks in Slot.js and I didn't see any exceptions get thrown.